metaflow/plugins/argo/argo_workflows.py

```diff
@@ -1113,6 +1116,11 @@ def parallelism(self, parallelism):
         self.payload["parallelism"] = int(parallelism)
         return self
 
+    def ttl_strategy(self, ttl_strategy={}):
+        if ttl_strategy != {}:
+            self.payload["ttlStrategy"] = ttl_strategy
+        return self
```

Comment:
more generic and possibly not in scope of this PR, but how is the situation handled where a user sets a malformed ttlStrategy, and Argo (I would assume?) rejects the workflow. Is the raised error explicit enough?

Reply:
related to this feature as I believe the current setup allows to set arbitrary values for the ttlStrategy

Reply:
Good point.  I will try it out so we know.  I have a feeling the error from Argo is readable and we propagate it to the user in a good way.